docs/upgrade.md

```diff
@@ -0,0 +1,95 @@
+## Example upgrade from 4.16.x to 4.16.y
+### Starting conditions:
+- A compatible seed image with target cluster is created. Seed image has `4.16.y` OCP version. [Both target cluster and seed image has configured a shared container partition for IBU.](https://docs.openshift.com/container-platform/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html)
+- Spoke cluster is successfully deployed and configured using a `ProvisioningRequest`:  
+```bash
+$ oc get provisioningrequests.o2ims.provisioning.oran.org        
+NAME           AGE   PROVISIONSTATE   PROVISIONDETAILS
+cluster-name   37m   fulfilled        Provisioning request has completed successfully
+```
+- [OADP and LCA is installed on the spoke cluster. OADP backend is configured on the spoke cluster.](https://docs.openshift.com/container-platform/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-install-operators.html)
+```yaml
+# policytemplates/version_4.16.x/sno-ran-du/sno-ran-du-pg-v4-16-x-v1.yaml
+
+policies:
+- name: operator-subscriptions
+  manifests:
+    # LCA
+    - path: source-crs/LcaOperatorStatus.yaml
+    - path: source-crs/LcaSubscriptionNS.yaml
+    - path: source-crs/LcaSubscriptionOperGroup.yaml
+    - path: source-crs/LcaSubscription.yaml
+    # OADP
+    - path: source-crs/OadpOperatorStatus.yaml
+    - path: source-crs/OadpSubscriptionNS.yaml
+    - path: source-crs/OadpSubscriptionOperGroup.yaml
+    - path: source-crs/OadpSubscription.yaml
+- name: oadp-backend
+  manifests:
+    - path: source-crs/OadpSecret.yaml
+      patches:
+        - data: <FILL>
+    - path: source-crs/OadpDataProtectionApplication.yaml
+      patches:
+        - spec:
+            backupLocations:
+            <FILL>
+```
+### Steps:
+- Create new clustertemplates and policytemplates directories for the new release according to sample git-setup. File names, namespaces and names should match the new release version.
```

Comment:
The directories and template v1 could already exist if there have been new cluster deployment with the new OCP release (e.g. for the seed cluster). Can we add some words to cover that case as it's what we put into the sample git-setup.